ocaml/testsuite/tests/typing-local/exclave.ml

```diff
@@ -73,30 +69,30 @@ Error: This value escapes its region
 (* following we check error detection *)
 let foo x =
   let local_ y = "foo" in
-  [%exclave] (Some y)
+  exclave_ Some y
 [%%expect{|
-Line 3, characters 19-20:
-3 |   [%exclave] (Some y)
-                       ^
+Line 3, characters 16-17:
+3 |   exclave_ Some y
+                    ^
 Error: The value y is local, so cannot be used inside exclave
```

Comment:
This error might want to get reworded now: either "inside an exclave" or "inside exclave_" (or "inside an exclave_", for that matter)